app/components/Dashboard/Table.tsx

```diff
@@ -156,7 +156,7 @@ const Table = ({ applications }: Props) => {
         </tbody>
       </StyledTable>
 
-      <Modal id={withdrawRowId} />
+      <Modal id={withdrawId} />
```

Comment:
Adding some integration tests for our dashboard page, ensuring that we can withdraw an application (i.e. call the updateApplicationMutation with the expected input), would ensure what this bug isn't reintroduced. Happy to pair with you to guide you in writing that test if needed